CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 <!--next-version-placeholder-->
 
+## v0.6.1 (2023-08-02)
```

Comment:
Why is this part of your pull request?

Reply:
This was accidentaly added, because the feature branch initially started from the main branch. The changes were removed from this PR.